Dockerfile

```diff
@@ -1,6 +1,12 @@
 
 ###################
-FROM centos:8 as config-editor
+FROM registry.centos.org/centos/centos:8 as config-editor
```

Comment:
Is this a different image? Looks like it's just getting the same base image in a different location.

Reply:
Nevermind. Just saw your mention of pull/push quota issues from Dockerhub.